source/grid/grid_generator.cc

```diff
@@ -4554,14 +4613,14 @@ namespace GridGenerator
               }
 
             this_cell.material_id = cell->material_id();
+            if (copy_manifold_ids)
+              this_cell.manifold_id = cell->manifold_id();
             cells.push_back(this_cell);
```

Comment:
You can use `std::move` here as well.

Reply:
I somewhat dislike writing `std::move` when we have a trivial type (`CellData` contains a C-style array, a union, and an integer) since it doesn't do anything in that case. Would you mind if I left it as-is? I will change it if you feel strongly about this.

Reply:
Never mind, I confused it with `SubCellData`.